lib/index.js

```diff
@@ -142,8 +134,8 @@ bedrock.events.on('bedrock.ready', function() {
  *              a job, -1 for unlimited.
  * @param fn(job, callback) the function to call to execute the job.
  */
-api.define = function(type, options, fn) {
-  logger.info('defining job type', type);
+api.define = (type, options, fn) => {
+  logger.info(`defining job type ${type}`);
```

Comment:
Maybe worth putting quotes around the type?

Reply:
I have notice in other places we are using a colan like:
`defining job type: foo`

What do you think of that approach?  It seems cleaner to me.

Reply:
Either fine for me too.